CONTRIBUTING.md

```diff
@@ -34,7 +56,7 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
 ### Reporting Issues
 
 * Make sure you have a [GitHub account](https://github.com/signup/free)
-* Submit a [Github issue](https://github.com/samvera-labs/active_encode/issues/) by:
+* Submit a [Github issue](https://github.com/samvera/{{library}}/issues/) by:
```

Comment:
Looks like the script didn't replace correctly here.  This change should be reverted.

```suggestion
* Submit a [Github issue](https://github.com/samvera-labs/active_encode/issues/) by:
```